packages/suite/src/components/suite/asset-picker/components/AssetRow/AssetImage.tsx

```diff
@@ -0,0 +1,32 @@
+import { NetworkSymbolExtended, NetworkType, getDisplaySymbol } from '@suite-common/wallet-config';
+import { getContractAddressForNetworkSymbol } from '@suite-common/wallet-utils';
+import { AssetLogo, type AssetLogoProps } from '@trezor/components';
+
+type AssetImageProps = {
+    networkSymbol: NetworkSymbolExtended;
+    networkType: NetworkType;
+    contractAddress?: string;
+    symbol: string;
+    size?: AssetLogoProps['size'];
+};
+
+export function AssetImage({
+    networkType,
+    contractAddress,
+    networkSymbol,
+    symbol,
+    size = 40,
+}: AssetImageProps) {
+    return (
+        <AssetLogo
```

Comment:
For network icons use `CoinLogo`. Svg is bundled with app (offline app). For tokens this AssetLogo